packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFUIDelegateHostApi.m

```diff
@@ -88,6 +88,55 @@ - (void)requestMediaCapturePermissionForDelegateWithIdentifier:(FWFUIDelegate *)
                                                             decision));
                                                   }];
 }
+
+- (void)runJavaScriptAlertPanelForDelegateWithIdentifier:(FWFUIDelegate *)instance
+                                                 message:(NSString *)message
+                                                   frame:(WKFrameInfo *)frame
+                                       completionHandler:(void (^)(void))completionHandler {
+  [self runJavaScriptAlertPanelForDelegateWithIdentifier:[self identifierForDelegate:instance]
+                                                 message:message
+                                                   frame:FWFWKFrameInfoDataFromNativeWKFrameInfo(
+                                                             frame)
+                                              completion:^(FlutterError *error) {
+                                                NSAssert(!error, @"%@", error);
+                                                completionHandler();
+                                              }];
+}
+
+- (void)runJavaScriptConfirmPanelForDelegateWithIdentifier:(FWFUIDelegate *)instance
+                                                   message:(NSString *)message
+                                                     frame:(WKFrameInfo *)frame
+                                         completionHandler:(void (^)(BOOL))completionHandler {
+  [self runJavaScriptConfirmPanelForDelegateWithIdentifier:[self identifierForDelegate:instance]
+                                                   message:message
+                                                     frame:FWFWKFrameInfoDataFromNativeWKFrameInfo(
+                                                               frame)
+                                                completion:^(NSNumber *isConfirmed,
+                                                             FlutterError *error) {
+                                                  NSAssert(!error, @"%@", error);
+                                                  completionHandler(isConfirmed.boolValue);
```

Comment:
Since `NSAssert` is ignored in production, we should change this to

```suggestion
                                                  if (error) {
                                                    completionHandler(NO);
                                                  } else {
                                                    completionHandler(isConfirmed.boolValue);
                                                  }
```

We have an issue with another method that doesn't ensure this is always called.